projects/legacy/components/tag/tag.component.ts

```diff
@@ -104,7 +104,7 @@ export class TuiTagComponent {
     }
 
     protected get displayText(): string {
-        return this.editedText === null ? this.value : this.editedText;
+        return this.editedText ?? this.value;
     }
```

Comment:
This is not an equivalent replacement, if `this.editedText` is `undefined`. It's okay for us?

Reply:
@waterplea why we use `=== null` instead of `== null`?